zhaquirks/__init__.py

```diff
@@ -242,8 +243,10 @@ def __init__(self, *args, **kwargs):
     def _turn_off(self):
         self._timer_handle = None
         _LOGGER.debug("%s - Resetting motion sensor", self.endpoint.device.ieee)
-        self.listener_event(CLUSTER_COMMAND, 253, ZONE_STATE, [OFF, 0, 0, 0])
-        self._update_attribute(ZONE_STATE, OFF)
+        self.listener_event(
+            CLUSTER_COMMAND, 253, ZONE_STATUS_CHANGE_COMMAND, [OFF, 0, 0, 0]
```

Comment:
That `253` magic number seems to be the tsn value:
https://github.com/zigpy/zigpy/blob/be5b2914a12ac6a19af80c5a83dab40d7957aa01/zigpy/zcl/__init__.py#L374

I believe that we can use `None` here.

Reply:
> That 253 magic number seems to be the tsn value

Yeah, not sure why ``253``/``254`` are used here in the first place? Seems to be unused. They are basically the upper limit though, so maybe there's a reason?

I'm not sure if we really want to use `None` though, as that's invalid and might break future stuff that (for some reason) uses the TSN?

Reply:
I'd probably leave the TSN values like this now, but @dmulcahey, do you know why those values were chosen for sending the "zone status change notification" command (from quirks to ZHA)?

Reply:
Just for future reference, waxman is using `None` as `tsn` (and `ZONE_STATE`):
https://github.com/zigpy/zha-device-handlers/blob/dda8d8a0e99692813f34d7e0518eecfdd7a02ece/zhaquirks/waxman/leaksmart.py#L56-L58